app/src/main/java/de/ph1b/audiobook/features/settings/dialogs/SortingModeDialogFragment.kt

```diff
@@ -0,0 +1,46 @@
+package de.ph1b.audiobook.features.settings.dialogs
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import com.afollestad.materialdialogs.MaterialDialog
+import de.ph1b.audiobook.R
+import de.ph1b.audiobook.features.bookOverview.list.BookComparator
+import de.ph1b.audiobook.injection.App
+import de.ph1b.audiobook.injection.PrefKeys
+import de.ph1b.audiobook.persistence.pref.Pref
+import javax.inject.Inject
+import javax.inject.Named
+
+/**
+ * Dialog for picking the UI theme.
```

Comment:
This lies

Reply:
You caught my copy-paste!

Reply:
Following clean code, you can delete this comment and the one where you copied too ;)